pkg/authorization/api/register.go

```diff
@@ -59,6 +77,52 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 
 		&RoleBindingRestriction{},
 		&RoleBindingRestrictionList{},
+	}
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		append(types,
+			&unversioned.Status{}, // TODO: revisit in 1.6 when Status is actually registered as unversioned
```

Comment:
@soltysh the API server won't start. They should become real resources and then we can remove this.